guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -73,7 +73,7 @@ ifndef::orcharhino[]
 +
 For more information on GPOs, see {RHELDocsBaseURL}9/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#how-sssd-interprets-gpo-access-control-rules_applying-group-policy-object-access-control-in-rhel[How SSSD interprets GPO access control rules] in _Integrating RHEL systems directly with Windows Active Directory (RHEL{nbsp}9)_.
 endif::[]
-.. If your {ProjectServer} is running in an IPV6-only network, set the `lookup_family_order` option in the `[domain/_ad.example.com_]` section of the `/etc/sssd/sssd.conf` file:
+.. If your {ProjectServer} runs on RHEL{nbsp}9.6 and earlier or RHEL{nbsp}10.0 and if it runs in an IPv6-only network, set the `lookup_family_order` option in the `[domain/_ad.example.com_]` section of the `/etc/sssd/sssd.conf` file:
```

Comment:
Perhaps change the order to first mention IPv6-only, given those networks are less common. Most readers already know to skip this and don't need to parse the complex version constraint.

Reply:
Applied.